MekHQ/src/mekhq/gui/RepairTab.java

```diff
@@ -856,7 +857,7 @@ public void refreshTaskList() {
 
     public void refreshTechsList() {
         int selected = techTable.getSelectedRow();
-        ArrayList<Person> techs = getCampaign().getTechs(true, null, false, false);
+        List<Person> techs = getCampaign().getTechs(true, null, true, false);
```

Comment:
A comment explaining the unnamed arguments would be nice. Hard to say what this does outside of an IDE (and even within you have to hover).

Reply:
Ah, within Idea it shows all the variable names inline if you use values like true, null, "", or false.
I'll add a comment.

Reply:
![image](https://user-images.githubusercontent.com/39067288/76250199-36b9b280-621b-11ea-8f7f-e6fd08e693c5.png)